server/prisma/migrations/20240711013942_/migration.sql

```diff
@@ -0,0 +1,36 @@
+/*
+  Warnings:
+
+  - You are about to drop the `Token` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "Token" DROP CONSTRAINT "Token_userId_fkey";
+
+-- DropTable
+DROP TABLE "Token";
+
+-- CreateTable
+CREATE TABLE "Track" (
+    "id" SERIAL NOT NULL,
+    "userId" INTEGER NOT NULL,
+    "spotifyId" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
+    "artists" TEXT NOT NULL,
+    "album" TEXT NOT NULL,
+    "previewUrl" TEXT,
+    "features" JSONB NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "Track_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Track_spotifyId_key" ON "Track"("spotifyId");
```

Comment:
Subtle suggestions:

SQL has some naming conversions.  
1. For table name, people usually use the Pascal case, e.g, TableName, MusicTrack, etc. "Track" is good, but I would suggest using "SpotifyMusicTracks" to avoid confusion.
2. For column name, the conventional way is using the Snake case, e.g., "user_id", "created_at", "updated_at", "track_spotify_id_key", etc.
3. SQL command and keywords are usually in all upper case.  You already did it great.

Reply:
Okay, I will update this thanks for your feedback